app/assets/src/components/views/discovery/mapping/CircleMarker.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import BasicPopup from "~/components/BasicPopup";
+import cs from "./circle_marker.scss";
+
+class CircleMarker extends React.Component {
+  render() {
+    const { size, onClick, hoverContent } = this.props;
+
+    const circleBody = (
+      <circle
+        cx="50%"
+        cy="50%"
+        // Don't let edges get cut off in the viewBox
+        r={Math.max(size - 4, 1) / 2}
```

Comment:
Are edges size 4? Can you make it into a property?

Reply:
👍 I'm not sure why it was around 4 but lower than that and I noticed some edge distortion 
![Screen Shot 2019-04-18 at 11 07 47 AM](https://user-images.githubusercontent.com/5652739/56382189-aff78380-61cb-11e9-8651-b0dd7138a51d.png)